biz.aQute.bndlib/src/aQute/bnd/service/result/FunctionWithException.java

```diff
@@ -21,8 +20,10 @@
 		return t -> {
 			try {
 				return apply(t);
+			} catch (RuntimeException e) {
+				throw e;
 			} catch (Exception e) {
-				throw Exceptions.duck(e);
+				throw new RuntimeException(e);
```

Comment:
The `duck` and `throwsUnchecked` methods need to be moved from this interface (where they are public methods) to some class in this package where they can be package private. Then the code above can use this `duck` method.

Reply:
This is an interface, I can't see a way how that can become private?